chocolatey/plugins/modules/win_chocolatey.py

```diff
@@ -80,6 +80,12 @@
     type: bool
     default: no
     version_added: '2.1'
+  remove_dependencies:
+    description:
+    - Remove a package's dependencies on uninstall.
+    type: bool
+    default: no
+    version_added: '2.10'
```

Comment:
This should reflect the collection version that the feature was added and not the Ansible version.

Reply:
Should this be changed to 1.0.3 to align with the next release?